frontend/src/i18n/en-US.json

```diff
@@ -220,13 +219,14 @@
   "vaultDetails.actions.updatePermissions.reload": "Reload",
   "vaultDetails.actions.editVaultMetadata": "Edit Vault Metadata",
   "vaultDetails.actions.downloadVaultTemplate": "Download Vault Template",
-  "vaultDetails.actions.showRecoveryKey": "Show Recovery Key",
+  "vaultDetails.actions.displayRecoveryKey": "Show Recovery Key",
   "vaultDetails.actions.archiveVault": "Archive Vault",
   "vaultDetails.actions.reactivateVault": "Reactivate Vault",
-  "vaultDetails.error.paymentRequired": "Your Cryptomator Hub license has expired or you exceeded the number of licensed seats. Please inform a Hub administrator renew or upgrade the license.",
+  "vaultDetails.actions.claimOwnership": "Become Vault Owner",
```

Comment:
Any reason why you changed this? I still prefer "Claim Ownership". It doesn't have to be a direct translation, I would even argue that English is the source.

Reply:
Reverted in 79d980d37177daab6df0ff4cc75b24a85da2d41b